.node-version

```diff
@@ -0,0 +1 @@
+18
```

Comment:
Let's use node20

Reply:
Thanks for the reminder, updated.

Reply:
/cc @sodatea WDYT? Node 20 will be LTS in few days.